### ISSUE TITLE
Reset Battle Encounters Back to predefined state

### ISSUE THREAD

**Issue by DrkNinja** [MMetze/DMHelper-Tracker#64]
**Discord User:**
Grazyn

**Describe the bug:**
 Nothing happens when trying to end a battle encounter. Does not reset encounter.

**DMHelper version:**
1.6 

**To Reproduce:**
1. Press end battle, the summary table comes up
2. Asks to confirm battle complete
3. Click yes 

**Expected behavior:**
Resets encounter for future use .

**Screenshots:**
N/A

**Desktop (please complete the following information):**

**Additional context:**

> [11:13 AM]  DMHelper: That wont make a difference, tbh. In 1.6, if you run a battle and the need to start fresh, you'll need to create a new encounter. The integration of the battle dialog into the main win dr ow has left a few gaps, "ending" a battle is one of them.

**Comment by dm-helper** [comment]
Has been "fixed" in that with the Ribbon UI, the "End Battle" button will disappear. There will still be a button to "View Statistics", but there will not be a way to end a battle per se other than to delete the battle from the campaign tree.

**Comment by DrkNinja** [comment]
I really think there should be a reset encounter button as a just in case measure.

**Comment by dm-helper** [comment]
Agreed that does seem reasonable, just reopened it to think about it. 

Question is: to what status does an encounter get reset? Basically just back to having only the characters in the middle of the map? What is the real use case? Only as an emergency brake? Then you could always just delete the battle and add a new one...
